src/main/java/org/openrewrite/maven/AbstractRewriteMojo.java

```diff
@@ -226,9 +226,14 @@ protected ResultsContainer listResults(ExecutionContext ctx) throws MojoExecutio
             List<Validated.Invalid<Object>> failedValidations = validations.stream().map(Validated::failures)
                     .flatMap(Collection::stream).collect(toList());
             if (!failedValidations.isEmpty()) {
-                failedValidations.forEach(failedValidation -> getLog().error(
-                        "Recipe validation error in " + failedValidation.getProperty() + ": " +
-                        failedValidation.getMessage(), failedValidation.getException()));
+                failedValidations.forEach(failedValidation -> {
+                    if (getLog().isDebugEnabled()) {
+                        getLog().debug("Recipe " + recipe + " invalid");
+                    }
+                    getLog().error(
+                            "Recipe validation error in " + failedValidation.getProperty() + ": " +
+                                    failedValidation.getMessage(), failedValidation.getException());
```

Comment:
I wonder if instead we should qualify the property with the recipe name prefix here; since running with debug mode enabled might be uncommon. What are your thoughts @joanvr ?